runtime/smithy-client/common/src/aws/smithy/kotlin/runtime/client/config/HttpChecksumClientConfig.kt

```diff
@@ -0,0 +1,40 @@
+package aws.smithy.kotlin.runtime.client.config
```

Comment:
correctness: missing license at the top. same applies to `Concurrency.kt`

Reply:
Thanks, I think we need to adjust our Ktlint for be more stringent when it comes to license headers